### PR TITLE
Feature/rename renew to add bundles

### PR DESCRIPTION
**Pull request by NikolasStelmakh**

### CODE REVIEW

File: server/constants.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
 const ACCOUNT_TYPES = {
   register: 'register',
-  renew: 'renew'
+  renew: 'renew',
+  // todo: fix db rules before setting custom value or create separate table for ACCOUNT_TYPES (with enum ids)
```
Comment (andreyvEze) [inline server/constants.js]: I think we don't need this comment

File: server/process-events.js [hunk 2]
```diff
@@ -114,8 +114,9 @@ const renewaddress = async (address, tpid, walletActor = '', walletPermission =
     }]
     options.actor = walletActor
   }
-  return fio.renewAddress({
+  return fio.addBundlesToAddress({
     address,
+    bundleSets: 1,
```
Comment (andreyvEze) [inline server/process-events.js:119]: create const for `1`

File: server/process-events.js [hunk 3]
```diff
@@ -305,10 +306,10 @@ async function broadcastNewAccountOrRenew({
     fioAction = address ?
       await regaddress( account, owner_key, tpid, actor, permission ) :
       await regdomain( domain, owner_key, tpid )
-  } else if (type === ACCOUNT_TYPES.renew) {
-    fioAction = address ?
-      await renewaddress( account, tpid ) :
-      await renewdomain( domain, tpid )
+  } else if (type === ACCOUNT_TYPES.renew && !address) { // todo: address check could be removed when ACCOUNT_TYPES.renew and ACCOUNT_TYPES.addBundles becomes different
+    fioAction = await renewdomain( domain, tpid )
```
Comment (andreyvEze) [inline server/process-events.js:310]: let's change comment to be: 'use addBundles in case of crypto handle'

File: server/routes/wallet.js [hunk 4]
```diff
@@ -664,8 +664,9 @@ router.post('/public-api/renew', handler(async (req, res) => {
   }
 
   const addressArray = address.split('@')
-  const renewAccount = addressArray.length === 2
-  const type = renewAccount ? 'account' : 'domain'
+  const isAddBundlesToAddress = addressArray.length === 2
+  const type = isAddBundlesToAddress ? 'account' : 'domain'
+  const accountType = isAddBundlesToAddress ? ACCOUNT_TYPES.addBundles : ACCOUNT_TYPES.renew;
```
Comment (andreyvEze) [inline server/routes/wallet.js:669]: let's leave `ACCOUNT_TYPES.renew`

**Review by andreyvEze — APPROVED** [review]

**Review by ericbutz — APPROVED** [review]

**Pull request merged**